lib/core.js

```diff
@@ -266,37 +308,63 @@ export = {
 
     return {
       config: opts,
-      // can be called remotely.
-      auth (_pub: unknown, cb: Function) {
+      /**
+       * `auth` can be called remotely
+       * @param {unknown} _pub
+       * @param {Function} cb
+       */
+      auth (_pub, cb) {
         cb()
       },
-      address (scope?: ScopeStr) {
+
+      /**
+       * @param {ScopeStr=} scope
+       */
+      address (scope) {
         return api.getAddress(scope)
       },
-      getAddress (scope?: ScopeStr) {
+
+      /**
+       * @param {ScopeStr=} scope
+       */
+      getAddress (scope) {
         setupMultiserver()
-        return ms.stringify(scope) || null
+        return ms.stringify(scope) ?? null
```

Comment:
Oh yeah I didn't realize that you're still using 12 in mobile. I can mark this as a breaking change (major).

For Ahau I don't think you have a problem because secret-stack is so reliable that you probably don't need to update it any time soon.